src/mongo/db/storage/mmap_v1/btree/btree_test_help.cpp

```diff
@@ -186,9 +186,9 @@ string ArtificialTreeBuilder<OnDiskFormat>::expectedKey(const char* spec) {
 
     // parsing a long long is a pain, so just allow shorter keys for now
     unsigned long long num = strtol(spec + 1, &endPtr, 16);
```

Comment:
can't remove strtol usage here, because of we need to know pointer position to byte after digit.
parseNumFromString don't provide such possibility